src/gui/MainApplication.cpp

```diff
@@ -86,37 +86,36 @@ bool MainApplication::event(QEvent* event)
 }
 
 #ifdef LMMS_BUILD_WIN32
-// This can be moved into nativeEventFilter once Qt4 support has been dropped
-bool MainApplication::winEventFilter(MSG* msg, long* result)
+bool MainApplication::nativeEventFilter(const QByteArray& eventType,
+					void* message, long* result)
 {
+	if(eventType != "windows_generic_MSG")
+	{
+		return false;
+	}
```

Comment:
Short if, can make a single line
```suggestion
	if (eventType != "windows_generic_MSG") { return false; }
```